pkg/annotations/annotations.go

```diff
@@ -109,7 +109,7 @@ var AllAllowedAnnotations = []string{
 	DisableFIPSAnnotation,
 	// Keep in sync with
 	// https://github.com/opencontainers/runc/blob/3db0871f1cf25c7025861ba0d51d25794cb21623/features.go#L67
-	// Once runc 1.2 is released, we can use the `runc features` command to get this programatically,
+	// Once runc 1.2 is released, we can use the `runc features` command to get this programmatically,
```

Comment:
would you mind dropping this and rebaseing on https://github.com/cri-o/cri-o/pull/8088

Reply:
:heavy_check_mark: